### PR TITLE
PKTZ new broadband borehole

### PR DESCRIPTION
**Pull request by Thomas-Benson**
Opening new borehole broadband site `PKTZ`
 - Sensor Trillium Compact 120PH-2 (10213)
   - Borehole sensor depth 10m, orientation tbd
 - Datalogger Centaur (10955)
   - firmware version 4.8.5
 - Site location 38.672315S,  175.804593E,  514.959m
 - New streams PKTZ LC10 HH-12Z + LH-12Z

### CODE REVIEW

**Review by CBurton90 — APPROVED** [review]

**Review by mnaguit — COMMENTED** [review]
Thanks @Thomas-Benson !
- should we use `495m` for site/station elevation?
- just double checking on network to be `TP` (Taupo Volcano Seismic Network)?

**Comment by Thomas-Benson** [comment]
Hi @mnaguit! There was a 20m mismatch in elevation between the Centaur's clock and the co-located GNSS site, and I went with the GNSS elevation which should be better constrained. However, I'm expecting to get updated coordinates from the next site visit and will fix everything then.

And picked the Taupo network code based on proximity:
image

**Comment by mnaguit** [comment]
Thanks @Thomas-Benson

File: install/sensors.csv [hunk 1]
```diff
@@ -855,6 +855,7 @@ Nanometrics,Trillium Compact 120,617,HD71,10,0,,0,0,0,0,0,0,2017-12-14T00:39:45Z
 Nanometrics,Trillium Compact 120,619,HD50,10,0,,0,0,0,0,0,0,2015-11-17T22:46:00Z,2016-07-18T01:54:00Z
 Nanometrics,Trillium Compact 120,619,HD64,10,0,,0,0,0,0,0,0,2017-04-18T00:00:02Z,2019-06-27T01:02:00Z
 Nanometrics,Trillium Compact 120PH-2,10094,WTVZ,10,0,,0,0,0,0,0,0,2024-06-21T01:43:00Z,9999-01-01T00:00:00Z
+Nanometrics,Trillium Compact 120PH-2,10213,PKTZ,10,0,,0,10,0,0,0,0,2024-07-16T22:06:00Z,9999-01-01T00:00:00Z
```
Comment (mnaguit) [inline install/sensors.csv:858]: do we need to update the azimuth?

File: network/stations.csv [hunk 2]
```diff
@@ -1631,6 +1631,7 @@ PKGZ,EC,Pakihiroa,-37.8664,178.0805,405,,WGS84,2010-06-22T00:00:00Z,9999-01-01T0
 PKHE,XX,Pakihiroa,-37.859023554,178.093517735,200,,WGS84,1994-08-01T05:33:00Z,1994-12-14T03:38:00Z
 PKIS,SX,Pukaki Dam,-44.190967535,170.146940461,494,,WGS84,2001-06-14T00:00:00Z,2015-06-30T23:59:59Z
 PKRC,XX,Pukeroa,-38.480729926,175.540484557,600,,WGS84,2001-01-18T13:22:00Z,2001-04-13T15:35:00Z
+PKTZ,TP,Puketapu Road,-38.672315,175.804593,514.959,,WGS84,2024-07-16T22:06:00Z,9999-01-01T00:00:00Z
```
Comment (mnaguit) [inline network/stations.csv:1634]: coordinates and elevation should be the same with the entry in `sites.csv`?

File: install/sensors.csv (hunk 1 above)
Comment (Thomas-Benson) [reply install/sensors.csv:858]: Yes, I still need to update the orientation here once Conrad has had a chance to look at the surface sensor data

File: network/stations.csv (hunk 2 above)
Comment (Thomas-Benson) [reply network/stations.csv]: Forgot to add new coordinates to this file 🤦 updated now

File: install/sensors.csv (hunk 1 above)
Comment (mnaguit) [reply install/sensors.csv:858]: Thanks @Thomas-Benson , I'll approve once the azimuth is updated.

Comment (CBurton90) [reply install/sensors.csv:858]: @Thomas-Benson @mnaguit I've posted orientation results in the main ticket. Thanks.

**Comment by Thomas-Benson** [comment]
Borehole orientation updated, which I think should be the last change for this station

**Review by mnaguit — APPROVED** [review]

**Review by CBurton90 — APPROVED** [review]

**Pull request merged**